index.html

```diff
@@ -123,18 +123,26 @@ <h4 class="panel-title">
                         Save password in your browser
                     </label>
                   </div>
+                  <div class="checkbox">
+                    <label class="control-label" for="ssl">
+                      <input type="checkbox" id="ssl" ng-model="settings.ssl">
+                      Encryption. <strong>Strongly recommended!</strong> Need help? Check below.
+                    </label>
+                  </div>
                   <div class="checkbox" ng-show="settings.savepassword">
                     <label class="control-label" for="autoconnect">
-                        <input type="checkbox" id="autoconnect" ng-model="settings.autoconnect">
+                        <input type="checkbox" id="autoconnect" ng-model="settings.autoconnect"  ng-disabled="settings.useTotp">
                         Automatically connect
                     </label>
                   </div>
                   <div class="checkbox">
-                    <label class="control-label" for="ssl">
-                      <input type="checkbox" id="ssl" ng-model="settings.ssl">
-                      Encryption. <strong>Strongly recommended!</strong> Need help? Check below.
+                    <label class="control-label" for="useTotp">
+                      <input type="checkbox" id="useTotp" ng-model="settings.useTotp" ng-change="changeUseTOTP()">
+                      Use Time-based One-Time Password
```

Comment:
This probably needs an :information_source: link next to it to the weechat docs or some kind of explanation